www/src/components/accounts/ServiceAccounts.js

```diff
@@ -27,12 +28,21 @@ function ServiceAccount({user, next, update}) {
       window.location = '/'
     }
   })
+  const me = useContext(CurrentUserContext)
+
+  function handleImpressonateClick() {
```

Comment:
rename to `handleImpersonateClick`.  Also I think this should probably be a `useCallback` hook accepting mutation as a dependency, although I suspect it wouldn't actually matter

Reply:
Good catch, it's my GitHub Copilot's mistake :)
No need for a useCallback here imo.